front_end/panels/ai_chat/ui/AIChatPanel.ts

```
@@ -766,6 +773,31 @@ export class AIChatPanel extends UI.Panel.Panel {
     this.#chatView.addEventListener('manual-setup-requested', this.#handleManualSetupRequest.bind(this));
   }
 
+  /**
+   * Initialize MCP integration: connect and discover tools if enabled,
+   * and react to MCP config changes from Settings.
+   */
+  #setupMCPIntegration(): void {
+    const initAndRefresh = async () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)